src/desktop_notifier/backends/dbus.py

```diff
@@ -260,6 +260,7 @@ async def get_capabilities(self) -> frozenset[Capability]:
             Capability.TITLE,
             Capability.TIMEOUT,
             Capability.URGENCY,
+            Capability.ON_DISPATCHED,
```

Comment:
opt: I'm not sure if we need to add this as a capability if its not backend dependent at all but rather fired by DesktopNotifier itself. But then again, its consistent with other event handlers being listed here :)

Reply:
Exactly my thinking as well :+1: